webview-ui/src/components/chat/ApiRequestDetailsBlock.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { useTranslation } from "react-i18next"
+import { safeJsonParse } from "@roo/shared/safeJsonParse"
+import { vscode } from "@src/utils/vscode"
+import { ClineApiReqInfo, ClineMessage } from "@roo/shared/ExtensionMessage"
+
+interface ApiRequestDetailsBlockProps {
+	message: ClineMessage
+	icon: React.ReactNode
+	cost: number | undefined
+	apiRequestFailedMessage: string | undefined
+	apiReqStreamingFailedMessage: string | undefined
+}
+
+const ApiRequestDetailsBlock: React.FC<ApiRequestDetailsBlockProps> = ({
+	message,
+	icon,
+	cost,
+	apiRequestFailedMessage,
+	apiReqStreamingFailedMessage,
+}) => {
+	const { t } = useTranslation()
+
+	return (
+		<div className="roo-tool-use border-green-400/50 ">
+			<div
+				className="flex gap-4"
+				onClick={() => {
+					const apiInfo = safeJsonParse<ClineApiReqInfo>(message.text)
+					if (apiInfo?.request) {
+						vscode.postMessage({
+							type: "showApiRequestDetails",
+							details: apiInfo.request,
+						})
+					}
+				}}
+				title={t("chat:apiRequest.viewDetailsTooltip") ?? "View API Request Details"}>
```

Comment:
We don't need inline English fallbacks for translations - they'll just use what's in the English json file if a match isn't found.
```suggestion
				title={t("chat:apiRequest.viewDetailsTooltip")}>
```